cart.spec

```diff
@@ -122,6 +122,9 @@ ln %{?buildroot}%{carthome}/{TESTING/.build_vars,.build_vars-Linux}.sh
 %{carthome}/.build_vars-Linux.sh
 
 %changelog
+* Tue Oct 22 2019 Alexander Oganezov <alexander.a.oganezov@intel.com>
```

Comment:
one thing that isn't clear to me...if the cart version requires a mercury version, does that imply a cart minor version bump?